welcome/index.adoc

```diff
@@ -32,7 +32,7 @@ a|link:../admin_guide/index.html[*Administrator Guide*]
 
 These topics describe how to use OpenShift as an administrator.
 
-a|link:../dev_guide/index.html[*Developer Guide*]
+a|link:../registry_quickstart/developers.html[*Developer Guide*]
```

Comment:
> Was this intentional?

No. Good catch! Reverted.

I discarded the Welcome topic in favor of the distro package index page.